tests/test_fest.py

```diff
@@ -164,6 +176,10 @@ async def test_update_task(self, cblpytest: CBLPyTest, dataset_path: Path) -> No
             DocumentEntry("_default.tasks", "db1-list1-task1")
         ])
 
+        self.mark_test_step("Start the replicators")
```

Comment:
This should be updated in the header and test spec as one of the steps.

Reply:
I have changed this based on the test spec.